service/templates/admin.html

```diff
@@ -2252,6 +2252,10 @@ <h2 class="modal-title">[[modalTitle]]</h2>
 										<label>Shutdown Wait(seconds)</label>
 										<input type="number" ng-model="Settings.shutdown_wait_seconds">
 									</md-input-container>
+									<md-input-container class="control" style="width:45%;">
+										<label>Oldest Allowed CLI Version</label>
+										<input type="text" ng-model="Settings.oldest_allowed_cli_version">
+									</md-input-container>
```

Comment:
Is there a plan in place for adding this to the spruce UI as well? In the past things being in admin settings but not in spruce broke things, so a deploy of both at the same time might be necessary.

Reply:
@aldo-aguilar are there docs for adding new admin settings to the new UI?

Reply:
Thanks, I added it to the GQL schema here and created a UI [PR](https://github.com/evergreen-ci/ui/pull/1081)